vars/common-vars.yml

```diff
@@ -17,6 +17,7 @@ socok8s_libvirtuuid: "{{ socok8s_workspace }}/libvirt.uuid"
 suse_osh_registry_location: "docker.io"
 suse_openstack_image_version: "rocky-opensuse_15"
 suse_infra_image_version: "latest-opensuse_15"
+suse_osh_openstack_release: "rocky"
```

Comment:
why `suse`? isn't that osh specific?

Reply:
Initially, just to make it similar as other variables above.

And I think there might be a time it could be used by airship as well

Reply:
I dont' like the same but I don't have a better idea :)

Reply:
I can change it to `osh_openstack_release` if it would make you happy